iaso/api/setup_account.py

```diff
@@ -75,6 +76,9 @@ def create(self, validated_data):
             modules=account_modules,
             analytics_script=validated_data.get("analytics_script", ""),
         )
+        account_feature_flags = [feature_flag["code"] for feature_flag in ACCOUNT_FEATURE_FLAGS]
+        account.feature_flags.set(account_feature_flags)
+        account.save()
```

Comment:
```suggestion
        account_feature_flags = ["SHOW_HOME_ONLINE", "SHOW_BENEFICIARY_TYPES_IN_LIST_MENU", "SHOW_PAGES"]
        user.iaso_profile.account.feature_flags.set(account_feature_flags)
```

I'd rather hardcode `code`s here and remove `.save()` because it's not necessary here.